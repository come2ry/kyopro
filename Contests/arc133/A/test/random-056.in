9
10 6 2 3 9 7 4 10 3
