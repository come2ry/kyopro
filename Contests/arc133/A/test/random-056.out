1 0 2 3 9 7 4 1 0 3
